server/routerlicious/packages/services-core/src/tenant.ts

```diff
@@ -67,6 +67,15 @@ export enum KeyName {
 	key2 = "key2",
 }
 
+export enum EncryptionKeyVersion {
```

Comment:
These interfaces should remain stable. These names will change every year resulting in code changes. If anyone has a dependency on these names then their code may break if a change is made here. 
Better names can be "keyOld", "keyNew".
The key rotation process can be like this -
 keyOld is replaced by keyNew.
 KeyNew is replaced by the new key.

Reply:
I assume `EncryptionKeyVersion` will be append only when we get to a new year. And after we add a new enum, for example `key2024 = "2024"`, we will update in other places in `tenantManager` to let it know we are using a new version. I will add some comment and a central variable to make it more clear in the code.

If we use "keyOld" and "keyOld", they will be pointed to "2022" and "2023" this year, and "2023" and "2024" next year, which could potentially cause confusion.

Reply:
If we add a new entry in this Enum every year, then it should be fine if we don't remove old entries. Enum will grow at the rate of one entry per year which is also fine. 
However, some teams may want to change their keys twice a year. This common logic may not work for them.

Reply:
We can still keep it append only enum. And we can make `incomingKeyVersion` a function that other teams can overwrite. If other teams want to use another enum value, they can add a new value in this enum and use it elsewhere.